src/js/babel.js

```diff
@@ -75,10 +75,18 @@ export default class BabelCompiler extends CompilerBase {
       let presets = this.attemptToPreload(opts.presets, 'preset');
       if (presets && presets.length === opts.presets.length) opts.presets = presets;
     }
+    const output = babel.transform(sourceCode, opts);
+
+    let sourceMaps;
+    if (output.map) {
+      sourceMaps = output.map;
+      sourceMaps.sourcesContent && delete sourceMaps.sourcesContent;
```

Comment:
since `output.code` already being forwarded, no reason to contain duplicated code in sourcemap object.